testonly/hammer/hammer.go

```diff
@@ -327,23 +353,25 @@ func newHammerState(ctx context.Context, cfg *MapConfig) (*hammerState, error) {
 	return &hammerState{
 		cfg:            cfg,
 		start:          time.Now(),
-		prng:           rand.New(cfg.RandSource),
 		prevContents:   &prevContents,
 		smrs:           &smrs,
 		validReadOps:   &validReadOps,
 		invalidReadOps: &invalidReadOps,
 	}, nil
 }
 
-func (s *hammerState) performOperations(ctx context.Context, done <-chan struct{}) (uint64, error) {
+// TODO(mhutchinson): Remove hammerState from here - it allows access to too much global info
+// which makes reasoning about the behaviour too difficult.
```

Comment:
nit: This sentence is a bit dramatic :D Nothing should be **too** difficult to Martin :)

Reply:
There's nothing like a bit of **DRAMA** to make a comment more interesting. No?

Toned it down.